recipe/meta.yaml

```diff
@@ -11,18 +11,18 @@ source:
 
 build:
   number: 1
-  skip: True  # [win]
+  skip: True  # [win or py<35]
```

Comment:
conda-smithy knows how to use this to create the build matrix. The specification below will only be used at build time and not "config time."